Products/ZenUtils/CmdBase.py

```diff
@@ -90,10 +90,7 @@ class CmdBase(object):
     """
     Class used for all Zenoss commands
     """
-
-    doesLogging = True
-
-    def __init__(self, noopts=0, args=None):
+    def __init__(self, noopts=0, args=None, should_log=True):
```

Comment:
Want to preserve back compat if a subclass had set `doesLogging` to False.  You could set should_log to None and if set to something other than None it is used otherwise use old behavior.